src/SyncTrayzor/Pages/Settings/SettingsView.xaml

```diff
@@ -15,6 +15,7 @@
         MinWidth="430" MaxWidth="430"
         MinHeight="400" 
         SizeToContent="WidthAndHeight"
+        ResizeMode="CanMinimize"
```

Comment:
Allowing users to make this window shorter is intentional - it's annoyingly tall on smaller screens, and it scales nicely. I have a faint memory that the NoSizeBelowScreenBehaviour below may require this behaviour as well - I'd need to check.